langfuse/client.py

```diff
@@ -138,8 +139,7 @@ def create_trace():
 
             def create_span():
                 try:
-                    new_body = body.copy(update={"id": new_span_id})
-                    new_body = body.copy(update={"trace_id": new_trace_id})
+                    new_body = body.copy(update={"id": new_span_id, "trace_id": new_trace_id})
```

Comment:
Reason : 2 seperate spans were created from same trace because before this change when setting `trace_id` , `id` was set to None.

Reply:
good catch!